Tasks/NuGetCommandV2/nugetcustom.ts

```diff
@@ -56,9 +57,16 @@ export async function run(nuGetPath: string): Promise<void> {
             tl.debug(`All URL prefixes: ${urlPrefixes}`);
         }
         let authInfo = new auth.NuGetExtendedAuthInfo(new auth.InternalAuthInfo(urlPrefixes, accessToken, useCredProvider, false), []);
+        
+        const useCredProviderV2: boolean = quirks.hasQuirk(NuGetQuirkName.V2CredentialProvider);
```

Comment:
For consistency, with the `isCredentialProviderV2Enabled` added, let's keep the same logic as above:
```ts
const useCredProviderV2: boolean = ngToolRunner.isCredentialProviderV2Enabled(quirks) && credProviderPath;
```

Reply:
👍